wp-content/themes/citylimits/inc/post-social.php

```diff
@@ -0,0 +1,193 @@
+        $output .= sprintf(
+            $fb_share,
+            rawurlencode( get_permalink() ),
+            __( ucfirst( of_get_option( 'fb_verb' ) ), 'largo' )
+        );
+    }
+
+    if ( isset( $utilities['twitter'] ) && '1' === $utilities['twitter'] ) {
+        $twitter_share = '<span class="twitter"><a target="_blank" href="https://twitter.com/intent/tweet?text=%1$s&url=%2$s%3$s"><i class="icon-twitter"></i><span class="hidden-phone">%4$s</span></a></span>';
+
+        // By default, don't set a via.
+        $via = '';
+
+        // If there are coauthors, use a coauthor twitter handle, otherwise use the normal author twitter handle
+        // If there is a custom byline, don't try to use the author byline.
+        if ( function_exists( 'coauthors_posts_links' ) && !isset( $values['largo_byline_text'] ) ) {
+            $coauthors = get_coauthors( $post->ID );
+            $author_twitters = array();
+            foreach ( $coauthors as $author ) {
+                if ( isset( $author->twitter ) ) {
+                    $author_twitters[] = $author->twitter;
+                }
+            }
+            if ( count( $author_twitters ) == 1 && !empty($author_twitters[0]) ) {
+                $via = '&via=' . rawurlencode( largo_twitter_url_to_username( $author_twitters[0] ) );
+            }
+            // in the event that there are more than one author twitter accounts, we fall back to the org account
+            // @link https://github.com/INN/Largo/issues/1088
+        } else if ( empty($via) && !isset( $values['largo_byline_text'] ) ) {
+            $user =  get_the_author_meta( 'twitter' );
+            if ( !empty( $user ) ) {
+                $via = '&via=' . rawurlencode( largo_twitter_url_to_username( $user ) );
+            }
+        }
+
+        // Use the site Twitter handle if that exists and there isn't yet a via
+        if ( empty( $via ) ) {
+            $site = of_get_option( 'twitter_link' );
+            if ( !empty( $site ) ) {
+                $via = '&via=' . rawurlencode( largo_twitter_url_to_username( $site ) ) ;
+            }
+        }
+
+        $output .= sprintf(
+            $twitter_share,
+            // Yes, rawurlencode. Otherwise, the link will break. Use html_entity_decode to handle wordpress saving smart quotes as &#1234; entities.
+            rawurlencode( html_entity_decode( get_the_title(), ENT_QUOTES, get_option('blog_charset') ) ),
+            rawurlencode( get_permalink() ),
+            $via,
+            __( 'Tweet', 'largo' )
+        );
+    }
+
+    if ( isset( $utilities['email'] ) && '1' === $utilities['email'] ) {
+        $output .= sprintf(
+            '<span data-service="email" class="email share-button"><a href="mailto:?subject=%2$s%3$s&body=%4$s%3$s%0D%0A%5$s%20%0D%0A%6$s" target="_blank"><i class="icon-mail"></i> <span class="hidden-phone">%1$s</span></a></span>',
```

Comment:
What I get from this in the post body is:

> From City Limits: Video: Brooklyn’s Role in the ’18 Vote & What the Results Mean for the BoroKings County was center stage on Election Night 2018, helping to flip the State Senate and the U.S. House blue. What drove the electoral shift behind those results, and what will their impact be on the borough? We asked two veteran journalists.
> https://citylimits.test/2018/11/08/video-brooklyns-role-in-the-18-vote-what-the-results-mean-for-the-boro/

There's no spacing between the headline and the excerpt.

The body string there is `%4$s%3$s%0D%0A%5$s%20%0D%0A%6$s`:

- %4$s - `'From City Limits '`
- %3$s - post title
- %0D%0A - carriage return, line feed
- %5$s - excerpt
- %20%0D%0A - space, carriage return, line feed
- %6$s - permalink

Adding a `%20` before the first `%0D%0A` solves the lack of space. I'm not sure why it's needed, though.

Reply:
Good catch. https://github.com/INN/umbrella-citylimits/pull/135/commits/95e00a6dd865e096c312003ed78aabe9f0ae28dc adds in an extra space there, and also adds a comment with your notes about the order of that string for whoever has to modify it next 😅 

Why that extra space is required... I'd think it probably has to do with what app/email provider is rendering the body string. For example, `%0D%0A` is supposed to be a return line but Gmail (browser, at least) ignores it.